### ISSUE TITLE
Example sentences 

### ISSUE THREAD

**Issue by Pikakitten** [mvysny/aedict#836]
The small kanzi icon shows examples but when I open it up in aedict, often the examples disappear for some reason

**Comment by mvysny** [comment]
Reproduced, thanks! Let me check why Aedict3 finds no examples for 没す/もっす

**Comment by mvysny** [comment]
The problem is that the "Examples" tab mandates that the example sentence contains both "没す" and "もっす", that is, it will try to find an example sentence where the word 没す is used exactly with the reading もっす.

If you search for example sentences for 没す, Aedict will return 4 sentences, for example 月が没した。In all of those sentences 没suru is used, which is a word different than 没す.

I therefore believe that Aedict does the right thing and shows no sentences (since none of those four example sentences applies to 没す). I'd say that the problem is in Aedict Reader which actually shows incorrect example sentences; it's Aedict Reader that needs to be fixed.

**Comment by Pikakitten** [comment]
Yeah, it does seem like Aedict reader is the faulty one, it didn't occur to me there was an almost identical verb to 没す

Though on the example tab for ぼっする, it still only shows 1 of the examples even though Tatoeba examples has 3 instances (technically 4 but one is a duplicate)

My guess is the remaining sentences don't show up because they're not exactly ぼっする but would appear if there was an entry for ぼっす 

If possible, maybe examples for verbs like 愛す/愛する should show identical example tabs, since they are the same verb (assuming they have the same reading, I do think you're right that もっす shouldn't display examples of ぼっす(る))
After all, the Tatoeba search does allow for exact matches so if somebody wants specifically あいする instead of あいす、あいした or the like, they could use that

Thanks for your time and your efforts in developing this great app!
